apps/client/src/components/CouponsLIst.jsx

```diff
@@ -0,0 +1,58 @@
+export const CouponsList  = ()=> {
```

Comment:
this component should get prop called coupons

```suggestion
export const CouponsList  = ({coupons})=> {
```